ldk/javascript/src/types/oliveHelps/browser.d.ts

```diff
@@ -48,12 +48,15 @@ declare namespace Browser {
     windowId: number;
   }
 
+  export type ListenType = 'click' | 'input';
```

Comment:
I get a dummy question about the ListenType. Do we only listen to these two types? Should we discuss what need to be added to the list and what should not? I think there are many different types of events on Browser.

Reply:
Thats a great point. There are definitely other event types that can be listened to. These were the only two that have seemed to be requested so far. Heres a list of [event](https://developer.mozilla.org/en-US/docs/Web/Events) that we could potentially account for/add to this list